translations/base-fr.yaml

```diff
@@ -506,49 +541,75 @@ buildings:
         default:
             name: &painter Peintre
             description: &painter_desc Colorie entièrement la forme de gauche avec la couleur de droite.
+
+        mirrored:
+            name: *painter
+            description: Colorie entièrement la forme de gauche avec la couleur de gauche.
```

Comment:
That doesn't look right. Both inputs can't be on the left, can they?

Reply:
You right. I hope my correction is better than this one